### DISCUSSION TITLE
KYC clarification

### DISCUSSION THREAD

**Discussion by nemishzalavadiya-pirimid** [ONDC-Official/ONDC-FIS-Specifications#25] [General]
According to figma shared here
[figma link](https://www.figma.com/file/TS8ghV1vPnZ9iod3ApVIKp/Credit?type=whiteboard&node-id=0-1&t=NrxbvmBDkNHUGqKb-0)

In GST invoice based flow
- Adhaar via DigiLocker
- Udyam certificate will be fetched from DigiLocker

But as I checked, for Digilocker only Adhaar is mandatory, Udyam may not be present, please confirm can we use any other 3rd party to fetch required data.

Can we also add other required Business documents needed for KYC other than Udyam?

**Comment by 92shreyansh** [comment]
@Antriksh-Parmar

**Reply by Antriksh-Parmar** [reply]
@nemishzalavadiya-pirimid could you clarify "Adhaar is mandatory, Udyam may not be present"

**Reply by nemishzalavadiya-pirimid** [reply]
DigiLocker app requires Aadhaar for sign in, so aadhaar will be available with DigiLocker.

But Udyam need to be added by **DigiLocker User** himself, so if user have not added in app, Udyam will not be available in issued document.

**Reply by Antriksh-Parmar** [reply]
@nemishzalavadiya-pirimid it can still be pulled even if it's not added.

**Reply by nemishzalavadiya-pirimid** [reply]
thank for confimation.
